rolling/release-rhel-build.yaml

```diff
@@ -38,6 +39,8 @@ package_blacklist:
 - rc_dynamics_api  # Not yet generated for RHEL
 - rc_genicam_api  # Not yet generated for RHEL
 - rc_genicam_driver  # Not yet generated for RHEL
+- ros_ign_bridge  # ignition has no RPM packages for RHEL
+- ros_ign_gazebo  # ignition has no RPM packages for RHEL
```

Comment:
Similarly, should we add `ros_ign`, `ros_ign_gazebo_demos`, and `ros_ign_image`?